.env

```diff
@@ -1 +1,2 @@
-REACT_APP_BASE_URL="https://one-small-thing.herokuapp.com/graphql"
\ No newline at end of file
+# REACT_APP_BASE_URL="https://one-small-thing.herokuapp.com/graphql"
+REACT_APP_BASE_URL="https://3bb63f73-e6d2-4120-8a53-4f012668a561.mock.pstmn.io"
```

Comment:
Good job in adding the Mock API so we don't pass the max requests in the BE's api.